resources/clj-kondo.exports/org.cyverse/clojure-commons/clojure_commons/config.clj

```diff
@@ -0,0 +1,135 @@
+(ns clojure-commons.config)
```

Comment:
This directory contains the actual pared down macro definitions that I created. The ones underneath `.clj-kondo` were placed there automatically by `clj-kondo`.